libpod/container.go

```diff
@@ -436,6 +487,17 @@ func (c *Container) Init() (err error) {
 
 	// Save the OCI spec to disk
 	jsonPath := filepath.Join(c.bundlePath(), "config.json")
+	// If the OCI spec already exists, replace it
```

Comment:
Why do we need to replace the OCI Spec on reboot?

Reply:
Yeah, I think you're right, nothing should change after reboot so we can use the existing spec